docs/codeql/query-help/index.rst

```diff
@@ -0,0 +1,22 @@
+CodeQL query help
+-----------------
+
+View the query help for the queries included in the ``code-scanning``, ``security-extended``, and ``security-and-quality`` query suites for the languages supported by CodeQL.
+
+Each query help article includes: 
+
+- A summary of key metadata for the query.
+- Information about which query suites the query is included in.
+- A link to the query in the `CodeQL repository <https://github.com/gitub/codeql>`__
```

Comment:
```suggestion
- A link to the query in the `CodeQL repository <https://github.com/github/codeql>`__.
```

... or technically the CodeQL for Go repository. But I don't know if we need to mention that separately.